vignettes/advs.Rmd

```diff
@@ -0,0 +1,313 @@
+---
+title: "Creating a Metabolic ADVS ADaM"
+output: 
+  rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Creating a Metabolic ADVS ADaM}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+
+library(admiraldev)
+```
+
+# Introduction
+
+This article describes creating a vital signs ADaM for metabolic clinical trials.
+
+We advise you first consult the `{admiral}` [Creating a BDS Finding ADaM
+vignette](https://pharmaverse.github.io/admiral/articles/bds_finding.html).
+The programming workflow around creating the general set-up of an ADVS using
+`{admiral}` functions is the same. We focus on the most common endpoints and their derivations mainly found in metabolic trials to avoid repeating information and maintaining the same content in two places.
+
+# Programming Workflow
+
+* [Initial ADVS Set-up](#advs_start)
+* [Derive Additional Parameters (e.g. `BMI` for `ADVS`)](#derive_param)
+* [Derive Categorization Variables (`AVALCATx`, `BASECATx`)](#cat)
+* [Derive Criterion Variables (`CRITy`, `CRITyFL`, `CRITyFLN`)](#crit_vars)
+* [Remaining ADVS Set-up](#advs_end)
+
+## Initial ADVS Set-up {#advs_start}
+
+The following steps are to merge ADSL variables with the source data and derive 
+the usual ADVS analysis variables. The `{admiral}` [Creating a BDS Finding ADaM
+vignette](https://pharmaverse.github.io/admiral/articles/bds_finding.html)
+gives detailed guidance on all these steps.
+
+The only difference here would be the the additional parameters collected in 
+metabolic trials - for example:
+
+`VSTESTCD` | `PARAMCD` | `PARAM` | `PARAMN` | `PARCAT1` | `PARCAT1N`
```

Comment:
> Wondering whether it would make more sense to showcase this table as an R tibble, so that users can copy it if they wish? 
Agreed. I'll add a tibble instead of this table. 

As it stands I can't easily replicate the mappings shown here.
I didn't understand this statement